### PR TITLE
total_seconds() instead of seconds

### PR DESCRIPTION
**Pull request by RoiTabach**

### CODE REVIEW

**Comment by RoiTabach** [comment]
Docs changes:
Before:
image

image

After:
image


image

**Comment by RoiTabach** [comment]
Expand the unit test on get_suppressed_alerts , have all the examples be 24 hours.

File: docs/quickstart/send-slack-alerts.mdx [hunk 1]
```diff
@@ -380,7 +380,7 @@ models:
   - name: marketing_leads
     meta:
       alerts_config:
-        alert_suppression_interval: 24
+        alert_suppression_interval: 48
```
Comment (RoiTabach) [inline docs/quickstart/send-slack-alerts.mdx:383]:
```suggestion
        alert_suppression_interval: 24
```

File: docs/quickstart/send-slack-alerts.mdx [hunk 2]
```diff
@@ -394,7 +394,7 @@ models:
           - unique:
               meta:
                 alerts_config:
-                  alert_suppression_interval: 24
+                  alert_suppression_interval: 72
```
Comment (RoiTabach) [inline docs/quickstart/send-slack-alerts.mdx:397]:
```suggestion
                  alert_suppression_interval: 24
```

**Review by RoiTabach — COMMENTED** [review]
.

**Review by IDoneShaveIt — APPROVED** [review]
LGTM!

**Pull request merged**